app/config/extensions/console.neon

```diff
@@ -4,14 +4,18 @@ extensions:
 console:
 	url: http://eventigo.cz
 	commands:
-		- App\Modules\Newsletter\Console\CreateNewslettersCommand
+		- App\Modules\Newsletter\Console\CreateNewsletterCommand
+		- App\Modules\Newsletter\Console\RenderNewslettersCommand
 		- App\Modules\Newsletter\Console\SendNewslettersCommand
 		- App\Modules\Admin\Console\GeneratePasswordCommand
 		- App\Modules\Admin\Console\CrawlSourcesCommand
 
 services:
 	createNewsletterCommand:
-		class: App\Modules\Newsletter\Console\CreateNewslettersCommand
+		class: App\Modules\Newsletter\Console\CreateNewsletterCommand
+		tags: [kdyby.console.command]
```

Comment:
Tags are often redundant information, I recommend them not using at all like this.
https://www.tomasvotruba.cz/blog/2017/02/12/drop-all-service-tags-in-your-nette-and-symfony-applications/